share/benchmark/xtp_benchmark.in

```diff
@@ -46,15 +51,26 @@ def xxquit(what=''):
 
 def ReadEnergyFromHDF5(filename):
     orb = h5py.File(filename, 'r')
-    a = orb['QMdata']['BSE_singlet']['eigenvalues'].value
-    a.flatten()
+    a = orb['QMdata']['BSE_singlet']['eigenvalues'][()]
```

Comment:
It would be better if the HDF5 file is handle inside a context manager like:
```py
with h5py.File(filename, 'r') as orb:
    arr = orb['QMdata']['BSE_singlet']['eigenvalues'][()]
return arr.flatten()
```